### PR TITLE
Little-fix

### PR DESCRIPTION
**Pull request by microcai**
一些小修正

### CODE REVIEW

File: include/ucoro/awaitable.hpp [hunk 1]
```diff
@@ -224,15 +224,18 @@ namespace ucoro
...
-					// 由于 initial_supend 为 suspend_always
-					// 因此 如果不对 .detach() 的返回值调用 co_await
-					// 此协程将不会运行。
-					// 因此，在本对象析构时，协程其实完全没运行过。
-					// 正因为本对象析构的时候，协程都没有运行，就意味着
-					// 其实用户只是调用了 .detach() 并没有对返回值进行
-					// co_await 操作。
-					// 因此为了能把协程运行起来，这里强制调用 resume
-					current_coro_handle_.resume();
+					if (current_coro_handle_.promise().continuation_ == nullptr)
```
Comment (Jackarain) [inline include/ucoro/awaitable.hpp:227]: 请问添加此判断的依据是什么？

根据我的理解，看起来似乎是多余的判断，因为执行到析构函数，能进入 `if (current_coro_handle_)` 这个条件，则说明该协程要么就是已经完成执行（即：`current_coro_handle_.done()`），要么则是仅仅是 detach 的情况，则应该在这里 `resume`

`current_coro_handle_.promise().continuation_` 不为空的情况，即 `co_await coro.detach()`，还进入到这里，这种情况应该需要在这里添加调试断言，因为这可能是哪里出错了。

**Review by Jackarain — COMMENTED** [review]
`if (current_coro_handle_.promise().continuation_ == nullptr)` 
上面条件的理由不充足，似乎是在做防御式编程，如果有其它的依据，请给出解释

File: include/ucoro/awaitable.hpp (hunk 1 above)
Comment (microcai) [reply include/ucoro/awaitable.hpp:227]: 正如提交信息里写的，我也不知道为啥，总感觉得这样写才好。。。。

Comment (Jackarain) [reply include/ucoro/awaitable.hpp:227]: 既然没有明确的理由，那么这部分先撤回修改吧，保持不变，如果真的遇到问题，我们应该找到具体的原因，再去做相应的修改。

**Review by Jackarain — APPROVED** [review]
看起来已经没有问题
